### PR TITLE
doc: Add introduction of using Windows Guest OS

### PR DESCRIPTION
**Pull request by lirui34**
Add introduction of using Windows Guest OS

Signed-off-by: ruix.li <ruix.li@intel.com>

### CODE REVIEW

File: doc/tutorials/using_windows_as_uos.rst [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+.. csv-table::
+   :header: "Platform Model", "Bios Version", "Download Link"
+
+   "NUC7i7DNHE", "DNKBLi7v.86A.0052.2018.0808.1344", "`link <https://downloadcenter.intel.com/download/28886?v=t>`__"
+   "NUC7i5DNHE", "DNKBLi5v.86A.0060.2018.1220.1536", "`link <https://downloadcenter.intel.com/download/28885?v=t>`__"
+
+ACRN Service OS Setup
+*********************
+You may refer to the steps in :ref:`getting-started-apl-nuc` for
+Intel NUC to setup ACRN on KBL NUC. After following the steps in that guide,
```
Comment (deb-intel) [inline doc/tutorials/using_windows_as_uos.rst]: Intel NUC to setup ACRN on KBL NUC.


Change to:
Intel NUC to set up ACRN on the KBL NUC

File: doc/tutorials/using_windows_as_uos.rst [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+Update Kernel on KBL NUC
+========================
+   #. Copy new kernel image (bzImage) and it's modules to target machine::
+
+      # scp <your host>:$WORKDIR/build/arch/x86/boot/bzImage /boot/bzImage
+      # scp -r <your host>:$WORKDIR/build-rootfs/lib/modules/* /lib/modules/
+      # cp /boot/loader/entries/acrn.conf  /boot/loader/entries/acrngt.conf
+
+   #. Modify ``acrngt.conf`` to the content as given below:
+
```
Comment (deb-intel) [inline doc/tutorials/using_windows_as_uos.rst]: After you left-align the numbered list, please move the code block info to the left as well:
#. Modify ``acrngt.conf`` to the content as given below:
    .. code-block:: none
       title The ACRNGT Service OS
       linux /bzImage

File: doc/tutorials/using_windows_as_uos.rst [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+      .. code-block:: none
+
+          title The ACRNGT Service OS
+          linux /bzImage
+          options console=tty0 console=ttyS0 root=/dev/sda3 rw rootwait ignore_loglevel no_timer_check consoleblank=0 i915.nuclear_pageflip=1 i915.avail_planes_per_pipe=0x010101 i915.domain_plane_owners=0x011100001111 i915.enable_gvt=1 i915.enable_conformance_check=0 i915.enable_guc=0 hvlog=2M@0x1FE00000
+
+      .. note:: You should change ``/dev/sda3`` to your file system partition.
```
Comment (deb-intel) [inline doc/tutorials/using_windows_as_uos.rst]: Please left-align notes:
.. note:: You should change ``/dev/sda3`` to your file system partition.

File: doc/tutorials/using_windows_as_uos.rst [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+
+   #. ``reboot`` Service OS and select ``The ACRNGT Service OS`` from the boot menu to apply the updates
+      of ACRN kernel and hypervisor
+
+Create Windows 10 Image
+=======================
+Now we need to create a Windows 10 Image which need two steps:
+
+   #. Re-generate an ISO which with virtio-win drivers and Windows graphics drivers
```
Comment (deb-intel) [inline doc/tutorials/using_windows_as_uos.rst]: Left-align numbered lists:
#. Re-generated an ISO . . .

**Comment by acrnsi** [comment]
OK to verify

File: doc/tutorials/using_windows_as_uos.rst [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+*****************
+* Cursor is not visible with GVG-g local display.
+* Windows graphic driver version must be ``dch_win64_25.20.100.6444.exe``;
+  the latest version ``dch_win64_25.20.100.6577.exe`` has problem with ``KVMGT`` and can’t be installed correctly.
+
+Device configurations of acrn-dm command line
+*********************************************
+* *-s 3,ahci,hd:/root/img/win10.img*:
+  This is the hard disk to install windows 10 onto. Please remember the slot ID 3 here.
+  Please make sure it points to your win10 img path.
```
Comment (deb-intel) [inline doc/tutorials/using_windows_as_uos.rst]:
```suggestion
  Make sure that the slot ID 3 points to your win10 img path.
```

File: doc/tutorials/using_windows_as_uos.rst [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+* *-s 6,virtio-input,/dev/input/event4*:
+  This is to passthrough the mouse/keyboard to Windows via virtio.
+  Please change ``event4`` accordingly. You can use the following command to check the event node on your SOS::
+
+   <To get the input event of mouse>
+   # cat /proc/bus/input/devices | grep mouse
+
+* *-s 7,ahci,cd:/root/img/Windows.iso*:
+  This is the IOS image used to install Windows 10. It appears as a cdrom device.
+  Please remember the slot ID 7 here and make sure it points to your win10 ISO path
```
Comment (deb-intel) [inline doc/tutorials/using_windows_as_uos.rst]:
```suggestion
Make sure that the slot ID 7 points to your win10 ISO path.
```

File: doc/tutorials/using_windows_as_uos.rst [hunk 7]
```diff
@@ -0,0 +1,375 @@
...
+
+* *-s 8,ahci,cd:/root/img/virtio-win-0.1.141.iso*: This is another cdrom device
+  to install virtio windows driver later. Please make sure it points to your VirtIO ISO path.
+
+* *--ovmf /root/bios/OVMF.fd*:
+  Please make sure it points to your OVMF binary. Please make sure it points to your OVMF binary path.
```
Comment (deb-intel) [inline doc/tutorials/using_windows_as_uos.rst]:
```suggestion
  Make sure it points to your OVMF binary path.
```

File: doc/tutorials/using_windows_as_uos.rst [hunk 8]
```diff
@@ -0,0 +1,375 @@
...
+
+   <To get the input event of mouse>
+   # cat /proc/bus/input/devices | grep mouse
+
+* *-s 7,ahci,cd:/root/img/Windows.iso*:
+  This is the IOS image used to install Windows 10. It appears as a cdrom device.
+  Please remember the slot ID 7 here and make sure it points to your win10 ISO path
+
+* *-s 8,ahci,cd:/root/img/virtio-win-0.1.141.iso*: This is another cdrom device
+  to install virtio windows driver later. Please make sure it points to your VirtIO ISO path.
```
Comment (deb-intel) [inline doc/tutorials/using_windows_as_uos.rst]:
```suggestion
  to install the virtio Windows driver later. Make sure it points to your VirtIO ISO path.
```

File: doc/tutorials/using_windows_as_uos.rst [hunk 9]
```diff
@@ -0,0 +1,375 @@
...
+Run these commands on Service OS::
+
+   # swupd bundle-add kvm-host
+   # mkdir /root/img
+   # cd /root/img
+   # qemu-img create -f raw win10-ltsc-virtio.img 30G
+
+Install Windows 10
+------------------
+Currently ACRNGT OVMF GOP driver is not ready, a special VGA version is used to install Windows 10
```
[the remark this answers is not in the document]
Comment (lirui34) [reply doc/tutorials/using_windows_as_uos.rst]: > IMPORTANT: Do you know in what release the driver will be ready??

@chejianj Do you have any idea about this question ?

[the remark this answers is not in the document]
Comment (chejianj) [reply doc/tutorials/using_windows_as_uos.rst]: The plan is that OVMF ACRNGT GOP driver will be ready at the end of Q3

[the remark this answers is not in the document]
Comment (deb-intel) [reply doc/tutorials/using_windows_as_uos.rst]: Thank you @lirui34 and @chejianj!

**Pull request merged**